.github/workflows/commit.yaml

```diff
@@ -17,6 +17,56 @@ on:
   workflow_dispatch:
 
 jobs:
+  test:
+    name: "Run unit tests"
+    runs-on: ${{ matrix.runner.os }}
+    timeout-minutes: 90  # instead of 360 by default
+    strategy:
+      matrix:
+        runner:
+          - os: macos-latest
```

Comment:
in circleci, we only tested ubuntu. this also tests macos which saves devs like me who use it. if this fails, we can assume dev workflow is busted.